themes/base16.ini

```diff
@@ -0,0 +1,82 @@
+[base]
+default          = none
+unknown-token    = 1,bold
+commandseparator = none
+redirection      = none
+here-string-tri  = 14
+here-string-text = bg:11
+here-string-var  = 1,bg:11
+exec-descriptor  = 9,bold
+comment          = 8
+correct-subtle   = 12
+incorrect-subtle = 1
+subtle-separator = 12
+subtle-bg        = bg:10
+secondary        =
+; recursive-base   =
+
+[command-point]
+reserved-word  = 5
+subcommand     = 6
+alias          = 4
+suffix-alias   = 4
+global-alias   = 4,bg:11
+builtin        = 4
+function       = 4
+command        = 4
+precommand     = 6
+hashed-command = 4
+single-sq-bracket = 4
+double-sq-bracket = 4
+double-paren   = 5
+
+[paths]
+path          = 9
+pathseparator =
```

Comment:
Is something needed here?

Reply:
Or does this just imply a space?

Reply:
Actually I see now that every other theme has a space after `pathseparator =`. Though I never noticed any issue without it and I don't understand what pathseparator is supposed to do?

Reply:
me either 😆